### PR TITLE
Bump ember version for component to ship w/ layout

### PR DESCRIPTION
**Pull request by billpull**
Previous update https://github.com/ololabs/principled-forms/pull/16 ended up causing the addon to ship the component without a layout. Bumping ember to 3.6 (& addon-docs due to https://github.com/ember-learn/ember-cli-addon-docs/issues/200) fixed the issue of attaching the layout.

### CODE REVIEW

**Review by chriskrycho — DISMISSED** [review]
LGTM, maybe bump to 3.7 since you're already at it?

File: packages/ember-principled-forms/jsconfig.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```
Comment (tansongyang) [inline packages/ember-principled-forms/jsconfig.json:1]: **?**: This doesn't appear to be relevant to the commit. Why was it included?

**Review by tansongyang — APPROVED** [review]
**?**: What is the extra stuff in the last commit for? Doesn't appear to be strictly necessary for upgrading to 3.8?

File: packages/ember-principled-forms/jsconfig.json (hunk 1 above)
Comment (chriskrycho) [reply packages/ember-principled-forms/jsconfig.json:1]: Heh, this is autogenerated stuff from VS Code, I think possibly turned on by one of the packages in the Ember VS Code collection. I've never figured out how to keep it from being generated, and now it bit @billpull as well.

Comment (tansongyang) [reply packages/ember-principled-forms/jsconfig.json:1]: Ah, I see.
